packages/ui/src/components/composition/Lists/VideosListComponent/VideoListComponent.tsx

```
@@ -97,26 +99,31 @@ export const VideoListComponent: React.FC<VideoListComponentProps> = ({
+			{"error" in props ? (
 				<ErrorAlert />
-			) : isLoading ? (
+			) : "isLoading" in props ? (
 				<SkeletonList />
-			) : data?.length && !isLoading ? (
+			) : props.data.length === 0 ? (
 				<AddVideoText />
 			) : (
 				<div className="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 w-full">
-					{data?.map((video: any) => (
+					{props.data?.map((video: any) => (
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)